### PR TITLE
[Bulk] Validation: Fix missing alt text issues 3/22

### PR DESCRIPTION
**Pull request by RedZephyr13**
This PR fixes 70 missing alt text issues.

### CODE REVIEW

File: iis/develop/using-webmatrix/creating-a-content-feed-for-learning.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ In this article, we will describe the integration experience through conditions
 
 WebMatrix has a selection of interesting states, or **triggers**, that it captures events for. When these occur, WebMatrix looks at the set of feeds it is currently indexing and opens a **learning panel** on the side of the application if there are any relevant entries in the feed for that event:
 
-[![](creating-a-content-feed-for-learning/_static/image2.png)](creating-a-content-feed-for-learning/_static/image1.png)
+[![Screenshot that shows the Web Matrix. Utils dot j s tab is highlighted as well as the Java Script File. ERROR HERE PLZ ADD NOTE!](creating-a-content-feed-for-learning/_static/image2.png)](creating-a-content-feed-for-learning/_static/image1.png)
```
Comment (Roudabeh1348) [inline iis/develop/using-webmatrix/creating-a-content-feed-for-learning.md:26]: web Matrix. Utils j s tab >Web Matrix dot j s tab

File: iis/extensions/url-rewrite-module/creating-rewrite-rules-for-the-url-rewrite-module.md [hunk 2]
```diff
@@ -129,11 +129,11 @@ Within the **Edit Rule** page, enter the following:
 
 Enter the name, pattern, and action as shown below:
 
-[![](creating-rewrite-rules-for-the-url-rewrite-module/_static/image19.jpg)](creating-rewrite-rules-for-the-url-rewrite-module/_static/image17.jpg)
+[![Screenshot that shows the Edit Inbound Rule pane. There are new entries for name, pattern and action.](creating-rewrite-rules-for-the-url-rewrite-module/_static/image19.jpg)](creating-rewrite-rules-for-the-url-rewrite-module/_static/image17.jpg)
 
 Enter the redirect URL as shown below:
 
-[![](creating-rewrite-rules-for-the-url-rewrite-module/_static/image23.jpg)](creating-rewrite-rules-for-the-url-rewrite-module/_static/image21.jpg)
+[![Screenshot that shows the Edit Inbound Rule pane. Under Action type, redirect has been selected and a Redirect U R L is entered.](creating-rewrite-rules-for-the-url-rewrite-module/_static/image23.jpg)](creating-rewrite-rules-for-the-url-rewrite-module/_static/image21.jpg)
```
Comment (Roudabeh1348) [inline iis/extensions/url-rewrite-module/creating-rewrite-rules-for-the-url-rewrite-module.md:136]: redirect> Redirect

File: iis/manage/provisioning-and-managing-iis/shared-configuration-and-remote-provisioning.md [hunk 3]
```diff
@@ -60,25 +60,25 @@ When read operation is performed, the underline configuration system is using th
 To work around the double-hop issue with configuration COM object, we need to change the DCOM configuration to use a specified account instead of caller's security context to enable the configuration COM object authenticating at the remote network share server with specified credential.  
 You can locate the IIS 7 configuration COM object – ahadmin under DCOM Config by using Component Services on the remote web server as illustrated in the following diagram:
 
-[![](shared-configuration-and-remote-provisioning/_static/image26.jpg)](shared-configuration-and-remote-provisioning/_static/image25.jpg)
+[![Screenshot that shows the ahadmin Properties dialog box. The launching user radio is selected.](shared-configuration-and-remote-provisioning/_static/image26.jpg)](shared-configuration-and-remote-provisioning/_static/image25.jpg)
```
Comment (Roudabeh1348) [inline iis/manage/provisioning-and-managing-iis/shared-configuration-and-remote-provisioning.md:63]: ahadmin Properties>a h admin Properties

File: iis/manage/provisioning-and-managing-iis/shared-configuration-and-remote-provisioning.md [hunk 4]
```diff
@@ -60,25 +60,25 @@ When read operation is performed, the underline configuration system is using th
...
 
 Right-click the ahadmin node, and select properties…, select the Identity tab, change the option to This User instead of The launching user, and specify a user account with the following permissions:
 
 1. Have change permissions on the shared configuration files on the network share server.
 2. Have read permission to redirection.config file stored under system32\inetsrv\config
 3. Member of local administrators group, otherwise you will have can't decrypt exception.
 
-[![](shared-configuration-and-remote-provisioning/_static/image28.jpg)](shared-configuration-and-remote-provisioning/_static/image27.jpg)
+[![Screenshot that shows the ahadmin Properties dialog box. The This user radio is selected with entries for User, Password, and Confirm Password.](shared-configuration-and-remote-provisioning/_static/image28.jpg)](shared-configuration-and-remote-provisioning/_static/image27.jpg)
```
Comment (Roudabeh1348) [inline iis/manage/provisioning-and-managing-iis/shared-configuration-and-remote-provisioning.md:71]: same as #63 ahadmin>a h admin

**Review by Roudabeh1348 — COMMENTED** [review]
I was not sure about my review for ahadmin but following your rules, I commented it would be better to be spaced out. :)

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**